plugins/admission/pkg/polaris/polaris.go

```diff
@@ -38,3 +42,31 @@ func GetPolarisReport(ctx context.Context, config polarisconfiguration.Configura
 	report.Contents = bytes
 	return report, nil
 }
+
+func GetPolarisValidateResults(kind string, decoder *admission.Decoder, req admission.Request, config polarisconfiguration.Configuration) (*validator.Result, error) {
+	var controller kube.GenericResource
+	var err error
+	if kind == "Pod" {
+		pod := corev1.Pod{}
+		err := decoder.Decode(req, &pod)
+		if err != nil {
+			return nil, err
+		}
+		if len(pod.ObjectMeta.OwnerReferences) > 0 {
+			logrus.Infof("Allowing owned pod %s/%s to pass through webhook", pod.ObjectMeta.Namespace, pod.ObjectMeta.Name)
+			return nil, nil
+		}
+		controller, err = kube.NewGenericResourceFromPod(pod, pod)
+	} else {
+		controller, err = kube.NewGenericResourceFromBytes(req.Object.Raw)
+	}
+	if err != nil {
+		return nil, err
+	}
+	// TODO: consider enabling multi-resource checks
```

Comment:
I'm curious to know more about what a multi-resource check would be - perhaps a deployment and accompanying HPA?

Reply:
Actually that `TODO` should not be there... i do not think there is a point we would have multiple resources validated at once.